build.sh

```diff
@@ -18,7 +18,7 @@ if [[ "$(uname)" == "Darwin" ]]; then
   tag_filter="$tag_filter,-dont-run-on-darwin,-scaladoc,-pdfdocs"
 fi
 
-SKIP_DEV_CANTON_TESTS=false
+SKIP_DEV_CANTON_TESTS=true
```

Comment:
@garyverhaegen-da is it what I am suppose to do to deactivate test mark with the tag `dev-canton-test` ?

Reply:
Yes.

Reply:
cc @moisesackerman-da @nickchapman-da